recipes/perf/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class Perf(ConanFile):
+    name = "perf"
+    description = "Linux profiling with performance counters"
+    topics = ("linux", "profiling")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://perf.wiki.kernel.org/index.php"
+    license = "GPL-2.0"
```

Comment:
The [source files](https://github.com/torvalds/linux/tree/b0f008551f0bf4d5f6db9b5f0e071b02790d6a2e/tools/perf) have different headers, but at [least one of them](https://github.com/torvalds/linux/blob/f3438b4c4e692b49b7dc2bab864d20381024be16/tools/perf/builtin-stat.c#L1) say that it is `GPL-2.0-only`

```suggestion
    license = "GPL-2.0-only"
```

Reply:
actually https://github.com/torvalds/linux/blob/master/COPYING says it is `"GPL-2.0 WITH Linux-syscall-note"`

Reply:
Yes, but that is the root COPYING file. Different Linux parts can have different licenses... 🤔